consensus/src/chained_bft/block_storage/sync_manager.rs

```diff
@@ -237,7 +231,7 @@ impl<T: Payload> BlockRetriever<T> {
     /// leader to drive quorum certificate creation The other peers from the quorum certificate
     /// will be randomly tried next.  If all members of the quorum certificate are exhausted, an
     /// error is returned
-    async fn retrieve_block_for_qc<'a>(
+    fn retrieve_block_for_qc<'a>(
         &'a mut self,
         qc: &'a QuorumCert,
```

Comment:
is this lifetime required any more?

Reply:
If you don't put it here, by the lifetime elision rules `self` and `qc` would have distinct lifetime vars. That does not seem to be a problem in this code though.